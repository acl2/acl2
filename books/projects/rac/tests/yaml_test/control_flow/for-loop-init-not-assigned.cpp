// RAC begin

int foo() {
  for (int i; i < 4; i++) {
  }
  return 0;
}
