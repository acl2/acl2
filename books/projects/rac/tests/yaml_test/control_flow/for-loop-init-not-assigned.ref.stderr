for-loop-init-not-assigned.cpp:4 (11-12): variable must be assigned directly
 4 |   for (int i; i < 4; i++) {
   |            ^
note: non initialized variable can introduce undefined behavior
